tests/autoyast/installation.pm

```diff
@@ -320,6 +321,12 @@ sub run {
         }
     }
 
+    # For xen, need set hard-disk as the boot device, then reboot and connect to existing VNC.
+    if (check_var('VIRSH_VMM_FAMILY', 'xen') && check_var('VIRSH_VMM_TYPE', 'hvm')) {
```

Comment:
Hmm, is it enough if only `check_var('VIRSH_VMM_TYPE', 'hvm')`

Reply:
For me with these two setting together to make the condition more accuracy for xen-hvm, so I prefer to use these two together as condition. Thanks.